geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala

```diff
@@ -29,29 +29,41 @@ class DelimitedTextConverterFactory extends SimpleFeatureConverterFactory[String
   val QUOTED_WITH_QUOTE_ESCAPE  = QUOTE_ESCAPE.withQuoteMode(QuoteMode.ALL)
 
   def buildConverter(targetSFT: SimpleFeatureType, conf: Config): DelimitedTextConverter = {
-    val format    = conf.getString("format") match {
-      case "DEFAULT"                  => CSVFormat.DEFAULT
+    val baseFmt = conf.getString("format").toUpperCase match {
+      case "CSV" | "DEFAULT"          => CSVFormat.DEFAULT
       case "EXCEL"                    => CSVFormat.EXCEL
       case "MYSQL"                    => CSVFormat.MYSQL
-      case "TDF"                      => CSVFormat.TDF
+      case "TDF" | "TSV" | "TAB"      => CSVFormat.TDF
       case "RFC4180"                  => CSVFormat.RFC4180
       case "QUOTED"                   => QUOTED
       case "QUOTE_ESCAPE"             => QUOTE_ESCAPE
       case "QUOTED_WITH_QUOTE_ESCAPE" => QUOTED_WITH_QUOTE_ESCAPE
       case _ => throw new IllegalArgumentException("Unknown delimited text format")
     }
+
+    val opts = {
+      import org.locationtech.geomesa.utils.conf.ConfConversions._
+      val o = "options"
+      var dOpts = new DelimitedOptions()
+      dOpts = conf.getBooleanOpt(s"$o.skip-header").map(s => dOpts.copy(skipHeader = s)).getOrElse(dOpts)
+      dOpts = conf.getIntOpt(s"$o.pipe-size").map(p => dOpts.copy(pipeSize = p)).getOrElse(dOpts)
+      dOpts
+    }
+
     val fields    = buildFields(conf.getConfigList("fields"))
     val idBuilder = buildIdBuilder(conf.getString("id-field"))
-    val pipeSize  = if(conf.hasPath("pipe-size")) conf.getInt("pipe-size") else 16*1024
-    new DelimitedTextConverter(format, targetSFT, idBuilder, fields, pipeSize)
+    new DelimitedTextConverter(baseFmt, targetSFT, idBuilder, fields, opts)
   }
 }
 
+case class DelimitedOptions(skipHeader: Boolean = false,
```

Comment:
it would actually be nice if skipHeader was an int - maybe this doesn't happen much, but for the microsoft tdrive data we have to drop the first 6 lines.